src/md/enc/metamodelrw.cpp

```diff
@@ -8780,7 +8780,7 @@ bool FilterTable::IsTokenMarked(
         //
         return true;
     }
-    return ( (*Get(rid)) & bitMarked ? true : false);
+    return ( ((*Get(rid)) & bitMarked) ? true : false);
```

Comment:
@Dmitry-Me The bar is not strictly defined. It is a heuristic e.g. to not require CLA for a trivial typo fix usually.